demo/serving/bentoml_serving/deeplabv3p_xception65_humanseg.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
thanks Chau, could we rename the folder from `bentoml_serving` to just `bentoml`? and rename notebook to `cloud-native-model-serving-with-bentoml.ipynb`?

Reply:
@parano renamed!